jupyter/opals_example.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Opals Example Notebook\n",
+    "This notebook is a short example of the Opals utilities provided by `adaptivefiltering`.\n",
+    "If you have not worked with `adaptivefiltering` before please have a look at the `demo.ipynb`.\n",
+    "\n",
+    "*This notebook uses a data set that is not shipped with `adaptivefiltering`, it can be run on <a href=\"https://ssc-jupyter.iwr.uni-heidelberg.de/hub/home\" target=\"_blank\">the SSC JupyterHub</a>*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import adaptivefiltering\n",
+    "from adaptivefiltering.asprs import asprs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = adaptivefiltering.DataSet(filename=\"data/uls_thingstaette.las\")\n",
```

Comment:
This should not have the `data/` prefix as we will not ship it in the data directory of `adaptivefiltering`.